### ISSUE TITLE
Analytics: Chart interval selector only has "By day" option on initial load

### ISSUE THREAD

**Issue by joshguss** [woocommerce/woocommerce-admin#5705]
**Describe the bug**
When going to an Analytics view that contains a chart with an interval selector (i.e. Overview), only the "By day" interval is available on initial load.

**To Reproduce**

1. Go to Analytics > Overview
2. Click on the interval selector dropdown (By day) for Charts

**Expected behavior**
The interval selector when clicked should display multiple interval options depending on the Date Range that you are viewing data for. To fix this, you can change the Date Range to a different Date Range from what was initially loaded (and then change back to the initially loaded date range if desired).

**Screenshots**
<img width="1261" alt="Screen Shot 2020-11-19 at 8 31 22 PM" src="https://user-images.githubusercontent.com/2687263/99813140-4fea6100-2b15-11eb-914f-bbfd6e2b2097.png">
<img width="1261" alt="Screen Shot 2020-11-19 at 8 32 07 PM" src="https://user-images.githubusercontent.com/2687263/99813151-537de800-2b15-11eb-99ee-3992aa0daacd.png">
<img width="1261" alt="Screen Shot 2020-11-19 at 8 32 31 PM" src="https://user-images.githubusercontent.com/2687263/99813158-55e04200-2b15-11eb-88ad-d8fdbf6c519f.png">

**Desktop (please complete the following information):**

* OS: macOS Catalina
* Browser: Chrome
* Version: 86.0.4240.198

**Comment by timmyc** [comment]
Hi @joshguss,
​
Thank you for taking the time to report this bug, we really appreciate your help. I can reproduce it on my end. 

I saw the same behavior on my site running WooCommerce Admin 1.7.3, using the month-to-date default, and the `day` option was only shown. When I manually set the date range to the same period, `week` was also shown.
​
We won’t be able to include this fix in the upcoming release due to the lower priority of this issue compared to others reported. We’re going to add it to our backlog so we can include it in our planning for one of our future releases.
